src/utils.js

```diff
@@ -3,6 +3,17 @@ import _ from 'lodash';
 const getYouTubeID = require('get-youtube-id');
 
 /**
+ * Returns true if the given elements collide. Only looks at
+ * their start_time and end_time.
+ */
+export function elementsCollide(e1, e2) {
+    if (e1.start_time <= e2.end_time && e1.end_time >= e2.start_time) {
```

Comment:
this can be simplified to `return e1.start_time <= e2.end_time && e1.end_time >= e2.start_time`